apps/user-portal/server.js

```diff
@@ -0,0 +1,82 @@
+'use strict';
```

Comment:
do we need to add license header here?

Reply:
This is a temporally file. But I have added the license header till we keep it.